Abonament.java

```diff
@@ -108,65 +110,35 @@ public static void schimbDeAbonament(String ID) {
         try (BufferedReader br = new BufferedReader(new FileReader("StareaContului.txt"))) {
             String line;
             while ((line = br.readLine()) != null) {
+                String newStare = "";
                 if (line.contains(ID)) {
-                    String newStare;
                     switch (abAles) {
                         case "Start 100":
                             newStare = "ID: " + ID + ", GB: " + Start100.gb + ", MIN: " + Start100.min;
-                            FileHelper.replaceLine("StareaContului.txt", line, newStare);
                             break;
                         case "Max 140":
                             newStare = "ID: " + ID + ", GB: " + Max140.gb + ", MIN: " + Max140.min;
-                            FileHelper.replaceLine("StareaContului.txt", line, newStare);
                             break;
                         case "Max 175":
                             newStare = "ID: " + ID + ", GB: " + Max175.gb + ", MIN: " + Max175.min;
-                            FileHelper.replaceLine("StareaContului.txt", line, newStare);
                             break;
                         case "Max 200":
                             newStare = "ID: " + ID + ", GB: " + Max200.gb + ", MIN: " + Max200.min;
-                            FileHelper.replaceLine("StareaContului.txt", line, newStare);
                             break;
                         case "Max 290":
                             newStare = "ID: " + ID + ", GB: " + Max290.gb + ", MIN: " + Max290.min;
-                            FileHelper.replaceLine("StareaContului.txt", line, newStare);
+                            break;
                     }
```

Comment:
aici tot poti sa folosesti acea structura noua la switch cu -> in loc sa scrii break de multe ori